exercises/nucleotide-count/example.R

```diff
@@ -0,0 +1,17 @@
+library(magrittr)
```

Comment:
Yeah I agree. Like you say the use of piping in R has become more common over time. Plus in many circles it's probably considered good practice since it improves readability. And ultimately the examples aren't really user facing anyway so it shouldn't matter too much.